### PR TITLE
Update docker.md Nginx section

### PR DESCRIPTION
**Pull request by maxdobeck**
Needed more details and small title change

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by maxdobeck** [comment]
small change @LoryApiFortress hope thats OK. Small title change to make it clearer sc sits next to the nginx container. And a clearer top level sentence highlighting what user should see

File: docs/secure-connections/sauce-connect/setup-configuration/docker.md [hunk 1]
```diff
@@ -155,9 +155,9 @@ $ cat /tmp/sc.ready | jq .
 }
 ```
 
-## Running the Sauce Connect Docker in the nginx Server
+## Running the Sauce Connect Docker with a nginx Server
 
-The `docker-compose.yaml` file below shows how to run Sauce Connect Docker in the nginx server.
+The `docker-compose.yaml` file below shows how to run Sauce Connect Docker in the nginx server. This is an example where you can use the tunnel while launching a Desktop or Web Mobile test, then go to the url 'nginx'. Sauce Connect Proxy will route you to the nginx container!
```
[the remark this answers is not in the document]
Comment (LoryApiFortress) [reply docs/secure-connections/sauce-connect/setup-configuration/docker.md:160]: @maxdobeck Might this work?

Comment (lysannep) [inline docs/secure-connections/sauce-connect/setup-configuration/docker.md:160]:
```suggestion
The `docker-compose.yaml` file below shows how to run Sauce Connect Docker in the nginx server. The example configuration allows you to use a tunnel while launching a Desktop or Web Mobile test, then go to the 'nginx' URL where Sauce Connect Proxy routes you to the nginx container.
```

**Review by lysannep — APPROVED** [review]

**Comment by lysannep** [comment]
@maxdobeck @LoryApiFortress Thank you!

**Pull request merged**